tests/trainers/README.md

```diff
@@ -0,0 +1,45 @@
+BCI tests modules
+=================
+
+Description
+-----------
+
+This directory contains modules and files that shall be copied into the container under test, for the execution of specific exercises and examples during the BCI validation .
+
+See the pytest_container/source/usage.rst documentation about the _copy to containers_.
+
+
+Tests for BCI python
+--------------------
+
+1. Communication examples: 
+
+Module to check communication capabilities of python functionalities. 
+
+The *get_file_www* function test that the python _wget_ [library](https://pypi.org/project/wget/) is able to fetch files from a webserver.
+
+We use the _download_ method to get a specific file from a remote url.
+
+Input parameters of this function are: (1) "URL/FIL" , (2) "DIR", where:
+
+- URL : the http remote url
+- FIL : the remote file to get
+- DIR : the directory in the container receiving FIL 
+
+Expected for this test: FIL present in DIR.
```

Comment:
All of this belongs into the docstring of `get_file_www` as then sphinx would be able to pick them up.

Reply:
Done